.huskyrc.js

```diff
@@ -0,0 +1,11 @@
+const { config } = require('@dhis2/cli-style');
+const husky = require(config.husky);
+
+const tasks = arr => arr.join(' && ');
+
+module.exports = {
+    hooks: {
+        ...husky.hooks,
+        'pre-commit': tasks(['yarn validate-commit']),
```

Comment:
Technically you don't need the `tasks` function here if you only want a single command.